be/src/exec/repeat_node.cpp

```diff
@@ -30,6 +30,7 @@ RepeatNode::RepeatNode(ObjectPool* pool, const TPlanNode& tnode,
                      const DescriptorTbl& descs)
     : ExecNode(pool, tnode, descs),
     _slot_id_set_list(tnode.repeat_node.slot_id_set_list),
+    _all_slot_id(tnode.repeat_node.all_slot_id),
```

Comment:
rename to `_all_slot_ids`.
And `tnode.repeat_node.all_slot_id` should be optional, so here you need to check whether this field is set.
Since this is a bug, you can just return ERROR if this field is not set.